src/include/execution/vm/bytecodes.h

```diff
@@ -172,6 +172,15 @@ namespace terrier::execution::vm {
                                                                                                                       \
   /* SQL type comparisons */                                                                                          \
   F(ForceBoolTruth, OperandType::Local, OperandType::Local)                                                           \
+  F(IsNull, OperandType::Local, OperandType::Local)                                                                   \
+  F(IsNotNull, OperandType::Local, OperandType::Local)                                                                \
```

Comment:
Reuse the `ValIsNull` and `ValIsNotNull` bytecodes rather than adding new ones - they're meant to do the same thing. Tweak the existing bytecodes to operate on primitive booleans.